client/app.js

```diff
@@ -3,10 +3,10 @@ import "./app.css";
 //component
 import ReduxTestComponent from "./Components/ReduxTestComponent/ReduxTextComponent";
 
-class App extends React.Component {
+export class App extends React.Component {
```

Comment:
Same thought as above. We should decide on how we want to export the components so it's consistent.

Reply:
It cannot run in the test without exporting components.